staging/src/k8s.io/kubectl/pkg/kuberc/kuberc.go

```diff
@@ -51,23 +54,31 @@ var (
 // arguments based on user's kuberc configuration.
 type PreferencesHandler interface {
 	AddFlags(flags *pflag.FlagSet)
+	ApplyPluginPolicy(*genericclioptions.ConfigFlags)
```

Comment:
Maybe `AddPluginPolicyWrapper` will better reflect what the function does. Also, please add a comment there, since that function will slightly stand out from the rest of the interface.

Reply:
fixed as per agreement in slack discussion